src/funding/venmo/config.jsx

```diff
@@ -19,12 +19,12 @@ export function getVenmoConfig(): FundingSourceConfig {
 
     layouts: [BUTTON_LAYOUT.HORIZONTAL, BUTTON_LAYOUT.VERTICAL],
 
-    eligible: ({ experiment }) => {
-      if (
-        experiment &&
-        (experiment.enableVenmo === false ||
-          experiment.venmoWebEnabled === false)
-      ) {
+    eligible: ({ experiment, shippingChange }) => {
+      if (experiment && experiment.enableVenmo === false) {
+        return false;
+      }
+
+      if (experiment.venmoWebEnabled === false && shippingChange) {
```

Comment:
I think this needs `experiment &&` at the start of the condition to fix the typecheck issue